src/content/self-hosted/administration/configuration.mdx

```diff
@@ -714,10 +714,14 @@ Configures [network policies](https://kubernetes.io/docs/concepts/services-netwo
 Disabled by default.
 
 - `blockedCIDRs`: Outgoing traffic to any cidr on this list will be blocked by the network policy (optional).
```

Comment:
Let's remove `blockedCIDRs` from here and the sample. It's deprecated by the new syntax